core/scss/layout/four-column/_molive.scss

```diff
@@ -0,0 +1,31 @@
+@charset "UTF-8";
+
+//
+// Molive
```

Comment:
If anyone has a problem with any of the layout names speak now or forever hold their peace.